src/javascript/app/pages/trade/digit_ticker.js

```diff
@@ -0,0 +1,101 @@
+const DIGIT_BLOCK_SIZE        = 36;
+const DIGIT_BLOCK_LEFT_OFFSET = 66;
+const DIGIT_BLOCK_CORRECTION  = 2;
+
+const DigitTicker = (() => {
+    let $container, source;
```

Comment:
generally should be in two separate lines,
```
let $container,
      source;
```

Reply:
not sure if it's more consistent to rename source to contract, and pass in the variable as proposal_open_contract, source seems a bit vague to me